src/solver/main.jl

```diff
@@ -129,6 +129,18 @@ julia> symbolic_solve(log(x+1)+log(x-1), x)
 julia> symbolic_solve(a*x^b + c, x)
 ((-c)^(1 / b)) / (a^(1 / b))
 ```
+
+### Evaluating output (converting to floats)
+If you want to evaluate the exact expressions found by `symbolic_solve`, you can do the following:
+```jldoctest
+julia> roots = symbolic_solve(2^(x+1) + 5^(x+3), x)
+1-element Vector{SymbolicUtils.BasicSymbolic{Real}}:
+ (-slog(2) - log(complex(-1)) + 3slog(5)) / (slog(2) - slog(5))
+
+julia> eval.(Symbolics.toexpr.(roots))
```

Comment:
we should not recommend eval for this. Instead, subtitute?

Reply:
`symbolics_to_float`?

Reply:
`substitute` works, `symbolic_to_float` doesnt though. 
```julia
julia> roots
1-element Vector{SymbolicUtils.BasicSymbolic{Real}}:
 (-slog(2) - log(complex(-1)) + 3slog(5)) / (slog(2) - slog(5))

julia> Symbolics.substitute(roots[1], Dict())
-4.512941594732059759689023145584186058252768936052415430071569066192919491762214 + 3.428598090438030380369414618548038962770087500755160535832807433942464545729382im

julia> Symbolics.symbolic_to_float.(roots)
ERROR: UndefVarError: `_x` not defined
```
Also, using `eval.(Symbolics.toexpr.(roots))` has never failed for me, why should we not use it?

Reply:
Aha, got it. Well, till symbolic_to_float is fixed id guess ill keep it like this. 
Also one point: i guess the user doesnt care if it has odd behavior when used in code or not since most likely itll be used in the REPL. Right?

Reply:
Try it again now that https://github.com/JuliaSymbolics/Symbolics.jl/pull/1267 is in.

Reply:
works nice